01-intro-to-r.Rmd

```diff
@@ -130,6 +131,21 @@ weight_kg <- 100
 
 What do you think is the current content of the object `weight_lb`? 126.5 or 220?
 
+### Saving your code
+
+Up to now, your code has been in the console. This is useful for quick queries
+but not so helpful if you want to revist your work for any reason.
+A script can be opened by pressing the <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + 
+<kbd>N</kbd>. 
+It it wise to save your script file immediately. To do this press <kbd>Shift</kbd> + 
```

Comment:
which operating system do you use? I thought that Ctrl + S is the usual shortcut to save a file.

Reply:
It is there, on the next line down. Would it better to move the CTRl to the next line also?